src/gc/gc.cpp

```diff
@@ -11373,6 +11374,13 @@ void gc_heap::adjust_limit_clr (uint8_t* start, size_t limit_size,
         }
         acontext->alloc_ptr = start;
     }
+    else  
+    {  
+        // If the next alloc context is right up against the current one it means we are absorbing the min  
+        // object, so need to account for that.  
+        acontext->alloc_bytes += (start - acontext->alloc_limit);  
```

Comment:
This is more of a question than a comment on this change, but it looks like this function is also used when we are allocating in the LOH (it's called from `a_fit_free_list_large_p`). If we're working in the LOH, shouldn't this code be modifying `alloc_bytes_loh` instead of `alloc_bytes`?

The managed API for getting allocated bytes seems to add `alloc_bytes_loh` and `alloc_bytes` together, so this wouldn't have any effect on that, but are we doing the right thing internally?